test/integration/test-dialect-sqlite.js

```diff
@@ -86,7 +86,7 @@ assert.equal(
 
 assert.equal(
 	dialect.escapeVal(new Date(d.getTime()), '-0400'),
-	"'2013-09-04T15:15:11.133Z'"
+	"'2013-09-04T19:15:11.133Z'"
```

Comment:
I suspect this test will fail on slower machines.
It should probably use http://sinonjs.org/releases/v4.4.2/fake-timers/ instead.